FilecoinFoundationWeb/filecoin-foundation
# [UXIT-1768] Clean Ecosystem Project Form

## 📝 Description
This PR cleans up the ecosystem project form based on feedback and suggestions from https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/693 - which we decided to close.

## 🛠️ Key Changes
The full list of changes can be found in the Notion [ticket](https://www.notion.so/filecoin/FF-Clean-ecosystem-project-form-1447631f2825809596aff4da11f98fbe?pvs=4). These updates improve the form without altering its core functionality.

The most important change was decoupling the form data formatting and the submit function. This will make our lives easier if we do decide to implement the form update step, or if we ever add unit tests to these functions.

## 📌 To-Do Before Merging
- [ ] Ask Jordan to review new content

## 🧪 How to Test
1. Go to `/ecosystem-explorer/project-form`
2. Submit the form and check that all the information is present in the right format

Two successful tests were made after implementing the changes:
1. https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/865
2. https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/866
